wpinet/CMakeLists.txt

```diff
@@ -9,47 +9,46 @@ file(GLOB wpinet_jni_src src/main/native/cpp/jni/WPINetJNI.cpp)
 
 # Java bindings
 if (WITH_JAVA)
-  find_package(Java REQUIRED)
-  find_package(JNI REQUIRED)
-  include(UseJava)
-  set(CMAKE_JAVA_COMPILE_FLAGS "-encoding" "UTF8" "-Xlint:unchecked")
+    find_package(Java REQUIRED)
+    find_package(JNI REQUIRED)
+    include(UseJava)
+    set(CMAKE_JAVA_COMPILE_FLAGS "-encoding" "UTF8" "-Xlint:unchecked")
 
-  set(CMAKE_JNI_TARGET true)
+    set(CMAKE_JNI_TARGET true)
 
-  file(GLOB_RECURSE JAVA_SOURCES src/main/java/*.java)
+    file(GLOB_RECURSE JAVA_SOURCES src/main/java/*.java)
 
-  add_jar(wpinet_jar ${JAVA_SOURCES} INCLUDE_JARS wpiutil_jar OUTPUT_NAME wpinet GENERATE_NATIVE_HEADERS wpinet_jni_headers)
+    add_jar(wpinet_jar ${JAVA_SOURCES} INCLUDE_JARS wpiutil_jar OUTPUT_NAME wpinet GENERATE_NATIVE_HEADERS wpinet_jni_headers)
 
-  get_property(WPINET_JAR_FILE TARGET wpinet_jar PROPERTY JAR_FILE)
-  install(FILES ${WPINET_JAR_FILE} DESTINATION "${java_lib_dest}")
+    get_property(WPINET_JAR_FILE TARGET wpinet_jar PROPERTY JAR_FILE)
+    install(FILES ${WPINET_JAR_FILE} DESTINATION "${java_lib_dest}")
 
-  set_property(TARGET wpinet_jar PROPERTY FOLDER "java")
+    set_property(TARGET wpinet_jar PROPERTY FOLDER "java")
 
-  add_library(wpinetjni ${wpinet_jni_src})
-  wpilib_target_warnings(wpinetjni)
-  target_link_libraries(wpinetjni PUBLIC wpinet)
+    add_library(wpinetjni ${wpinet_jni_src})
+    wpilib_target_warnings(wpinetjni)
+    target_link_libraries(wpinetjni PUBLIC wpinet)
 
-  set_property(TARGET wpinetjni PROPERTY FOLDER "libraries")
+    set_property(TARGET wpinetjni PROPERTY FOLDER "libraries")
 
-  target_link_libraries(wpinetjni PRIVATE wpinet_jni_headers)
-  add_dependencies(wpinetjni wpinet_jar)
-
-  install(TARGETS wpinetjni EXPORT wpinetjni)
+    target_link_libraries(wpinetjni PRIVATE wpinet_jni_headers)
+    add_dependencies(wpinetjni wpinet_jar)
 
+    install(TARGETS wpinetjni EXPORT wpinetjni)
 endif()
 
 if (WITH_JAVA_SOURCE)
-  find_package(Java REQUIRED)
-  include(UseJava)
-  file(GLOB WPINET_SOURCES src/main/java/edu/wpi/first/net/*.java)
-  add_jar(wpinet_src_jar
-  RESOURCES NAMESPACE "edu/wpi/first/net" ${WPINET_SOURCES}
-  OUTPUT_NAME wpinet-sources)
+    find_package(Java REQUIRED)
```

Comment:
Indentation seems inconsistent across this PR.  Some changes make what's inside if() statements go from 2 spaces to 4 (like this change).  Others add changes with 2 space indent?

Reply:
I tried to make the indentation consistent in individual files, but not across the whole project. The files sometimes have 2 spaces, and others have 4. I just changed mismatched blocks to whatever indentation was most used in the file. I could change the indentation to be consistent across all files, but we'd need to establish a consistent indentation.